src/pages/design/icons/index.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Icon Overview
+---
+
+import { graphql } from 'gatsby';
+
+import IconShowcase from '../../../examples/design/icons/IconShowcase.tsx';
+import IconShowcaseCode from '!!raw-loader!../../../examples/design/icons/IconShowcase.tsx';
+
+<CodeExample code={IconShowcaseCode}>
+  <IconShowcase />
+</CodeExample>
+
+### Usage
+
+Garden’s icon library is implemented using SVG. Icons come two sizes: 12px and 16px.
+The 16px icons can also be doubled to create 32px icons.
+
+#### Styles
+
+import IconStyles from '../../../examples/design/icons/IconStyles.tsx';
+import IconStylesCode from '!!raw-loader!../../../examples/design/icons/IconStyles.tsx';
+
+<CodeExample code={IconStylesCode}>
+  <IconStyles />
+</CodeExample>
+
+Icons also come in two styles: stroke and fill.
+While stroke icons are the default icons to use, fill icons are used to show selected states.
+
+#### 12px and 16px
+
+The default size is 16px, and should be used for the majority of cases.
+In certain instances where 16px icons will break the line height or will not fit in a small space, 12px icons should be used.
+
+<BestPractice>
+  <Do imageSource={props.data.defaultIconSize.childFile.childImageSharp.gatsbyImageData}>
+    <p>16px is the default icon size. Use 16px icons where possible.</p>
+  </Do>
+</BestPractice>
+
+<BestPractice>
+  <Do imageSource={props.data.smallerIconSize.childFile.childImageSharp.gatsbyImageData}>
+    <p>Use 12px icons for compact spaces where 16px icons can’t fit.</p>
+  </Do>
+</BestPractice>
+
+#### 32px
+
+32px icons are the scaled-up versions 16px icons. They are used for illustration only.
+
+<BestPractice>
+  <Do imageSource={props.data.illustrationIconSize.childFile.childImageSharp.gatsbyImageData}>
+    <p>Use 32px icons for illustration.</p>
+  </Do>
+</BestPractice>
+
+<BestPractice>
+  <Dont imageSource={props.data.uiIconSize.childFile.childImageSharp.gatsbyImageData}>
+    <p>Don’t use 32px icons for interactions.</p>
+  </Dont>
+</BestPractice>
+
+### Best practices
+
+#### Use tooltips with interactive icons
+
+<BestPractice>
+  <Do imageSource={props.data.tooltipLabelIcon.childFile.childImageSharp.gatsbyImageData}>
+    <p>Use tooltips to label icons when they are used as actions. </p>
```

Comment:
Double checking here: Is it just for actions? Are they used to simply identify things as well like concepts or navigation? 
![image](https://user-images.githubusercontent.com/90706608/156438461-254a2b2a-7017-4744-9627-e0ef0dc701e6.png)
To be inclusive of that, I'm thinking the section title should be "Include tooltips for interactive icons" and the image description to be "Use tooltips to clearly label interactive icons for accessibility." 
@bethanybeech or someone else who has been longer on the Garden team: that's the reason for these tooltips, correct? I feel like I heard Allison mention it's for accessibility.

Reply:
Sure, that makes sense to me.